package.json

```diff
@@ -38,7 +38,7 @@
     "predeploy": "yarn build",
     "deploy": "gh-pages -d dist/",
     "update:static": "cp src/wasm/multisig.wasm dist/multisig.wasm",
-    "start": "yarn update:static; parcel src/index.html",
+    "start": "yarn update:static; parcel -p 4000 src/index.html",
```

Comment:
Is the port really necessary here? Might screw up others who assume 1234 is the wallet

Reply:
See PR description:

> Someone in such a situation will very likely be running a different app using Parcel on port 1234, and will need to sign into Wallet using a different port. This switches the local Wallet port to 4000

If we want to avoid a tiny speed bump for the existing core Wallet devs, we should keep this unchanged. If we want to optimize for "anyone in the future trying to run everything locally", I think it makes sense to assume they might be running their main app using Parcel's default port, and to use something less common here.

Reply:
I tried adding `PORT=4000` to `.env.local`, but Parcel ignores the setting there

Reply:
if the goal is to support local devs using unmodified wallet, it makes sense to bundle whole thing and serve it without Parcel

Reply:
I completely agree but I don't have time for that right now